### PR TITLE
[Halo 3] Some Tag Definitions

### PR DESCRIPTION
**Pull request by matty45**

### CODE REVIEW

**Comment by Akarias** [comment]
A glance over, looks fine. 

Please complete the string versioning for styles etc. String versioning is a requirement for validation.

File: Tags/TagDefinitions/blofeld/objects/widgets/cloth.cpp [hunk 1]
```diff
@@ -28,10 +28,17 @@ namespace blofeld
...
 		{ _field_real, "default_distance*" },
 		{ _field_long_integer, "index1*" },
 		{ _field_long_integer, "index2*" },
 		{ _field_pad, "vmx_pad", 4 },
+
+		{ _field_version_less, _engine_type_haloreach, 3 },
+		{ _field_short_integer, "index1*" },
+		{ _field_short_integer, "index2*" },
+		{ _field_real, "default_distance*" },
+
```
Comment (LeeFoulger) [inline Tags/TagDefinitions/blofeld/objects/widgets/cloth.cpp:41]: Versions should be less above greater or equal

```suggestion
		{ _field_version_less, _engine_type_haloreach, 3 },
		{ _field_short_integer, "index1*" },
		{ _field_short_integer, "index2*" },
		{ _field_real, "default_distance*" },
		
		{ _field_version_greater_or_equal, _engine_type_haloreach, 4 },
		{ _field_real, "default_distance*" },
		{ _field_long_integer, "index1*" },
		{ _field_long_integer, "index2*" },
		{ _field_pad, "vmx_pad", 4 },
```

Comment (matty45) [reply Tags/TagDefinitions/blofeld/objects/widgets/cloth.cpp]: Should be fixed now!

**Comment by matty45** [comment]
Should these be moved to the behaviour sets according to assembly? https://i.imgur.com/ixP9c9g.png

**Pull request closed without merging**